lib/Mouse/Meta/Attribute.pm

```diff
@@ -256,12 +256,15 @@ sub install_accessors{
             my $generator = '_generate_' . $type;
             my $code      = $accessor_class->$generator($attribute, $metaclass);
             my $name      = $attribute->{$type};
-# TODO: do something for compatibility
-#            if( $metaclass->name->can($name) ) {
-#                my $t = $metaclass->has_method($name) ? 'method' : 'function';
-#                Carp::cluck("You are overwriting a locally defined $t"
-#                    . " ($name) with an accessor");
-#            }
+            my $is_method_defined = do {
```

Comment:
Does this check whether functions are stub or not?
Then I think variable names like `$is_not_stub` are more appropriate.

Reply:
I see, then I'll change the variable name.